src/apps/content-editor/src/app/components/Editor/PreviewMode/PreviewMode.js

```diff
@@ -33,11 +33,17 @@ export default function PreviewMode(props) {
       if (itemZUID) {
         const item = content[itemZUID];
 
-        let url = item?.web?.path
-          ? `${item.web.path}`
-          : `/-/instant/${item.meta.ZUID}.json`;
+        let url = "";
 
-        url = `${url}?_bypassError=true&__version=${props.version}`;
```

Comment:
Does this need to be extracted? Are there other code paths that need item URL determination?